subsys/net/ip/Kconfig

```diff
@@ -265,8 +265,11 @@ config NET_MAX_NEXTHOPS
 	  This determines how many entries can be stored in nexthop table.
 
 config NET_ROUTE_MCAST
-	bool
+	bool "Enable Multicast Routing / Forwarding"
+	default n
```

Comment:
The option is "off" by default so no need to mention it, you can just remove the default line.